packages/shared/components/boxes/CopyableBox.svelte

```diff
@@ -8,9 +8,7 @@
     export let value = ''
     export let isCopyable = true
     export let clearPadding = false
-    export let clearBoxPadding = false
     export let offset: number = undefined
-    export let classes = ''
```

Comment:
When removing `classes` and `clearBoxPadding` from here, I think that in the `KeyValueBox` component it should also be removed
This change would also affect all sites where `KeyValueBox` is used 😅